bisection.py

```diff
@@ -553,7 +553,7 @@ def output_abtest_result(self):
                                     targets=targets,
                                     output_json=args.output,
                                     devbig=args.devbig,
-                                    build_lazy=args.build_lazy,
+                                    build_lazy=True,
```

Comment:
There is no need to hardcode build_lazy here because you can specify it in run-bisection.sh.

Reply:
Will fix that in my PyTorch branch.